lighthouse-core/lib/page-functions.js

```diff
@@ -288,24 +292,26 @@ function getNodePath(node) {
     while (prevNode = node.previousSibling) {
       node = prevNode;
       // skip empty text nodes
-      if (node.nodeType === Node.TEXT_NODE && node.nodeValue.trim().length === 0) continue;
+      if (node.nodeType === Node.TEXT_NODE && (node.nodeValue || '').trim().length === 0) continue;
```

Comment:
`nodeValue` [technically](https://dom.spec.whatwg.org/#dom-node-nodevalue) can be null, so, fine